### ISSUE TITLE
[Bug] Migration of Observables into Cassandra is extremly slow

### ISSUE THREAD

**Issue by crackytsi** [TheHive-Project/TheHive#1333]
### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Debian
| OS version (client)       | 10
| TheHive version / git hash   | 4.0 RC2
| Package Type              | DEB

### Problem Description
Import of Observables in Cassandra is extremly slow.
By checking the CPU Usage, memory consum and IO-Wait time i can't see any clear limit that is the reason for the slow import.
It seems also especially importing of observables is extrem slow. 
I have cases with 2000 Observables that means it takes arround 4-5 hours to import that single case.

Here you can see a screenshot:
```
Tasks: 171 total,   1 running, 170 sleeping,   0 stopped,   0 zombie
%Cpu0  : 27.9 us, 10.5 sy,  0.0 ni, 60.1 id,  0.0 wa,  0.0 hi,  1.4 si,  0.0 st
%Cpu1  : 21.7 us, 13.0 sy,  0.0 ni, 63.9 id,  0.0 wa,  0.0 hi,  1.4 si,  0.0 st
%Cpu2  : 20.7 us,  8.3 sy,  0.0 ni, 70.3 id,  0.0 wa,  0.0 hi,  0.7 si,  0.0 st
%Cpu3  : 20.8 us,  9.0 sy,  0.0 ni, 69.9 id,  0.0 wa,  0.0 hi,  0.4 si,  0.0 st
KiB Mem:  33019092 total, 14386212 used, 18632880 free,   115900 buffers
KiB Swap:  3019772 total,        0 used,  3019772 free.   898936 cached Mem

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
 1903 cassand+  20   0  9.864g 8.332g  34152 S  82.8 26.5   7:22.84 java

```

### Possible Solutions
Unforunately I don't know a solution...

**Comment by nadouani** [comment]
Hello @crackytsi issue identified, @To-om is working on it, but it's basically related to constraint checking and locks.

**Comment by To-om** [comment]
This is fixed in 4.0.0-RC3.
